feature-libs/organization/order-approval/components/details/order-detail-permission-results/order-detail-permission-results.component.ts

```diff
@@ -1,6 +1,6 @@
 import { ChangeDetectionStrategy, Component } from '@angular/core';
-import { Order } from '@spartacus/core';
-import { OrderDetailsService } from '@spartacus/storefront';
+import { OrderDetailsService } from '@spartacus/order/components';
+import { Order } from '@spartacus/order/root';
```

Comment:
Since we release thiis in a minor, we have to use a dynamic system that imiports from the lib if it's loaded in the app or fall back to the deprecated code otherwise.  I don't know yet how this is done.

Let's dicuss.  Perhaps a follo-up ticket would be required to change how the organization lib depends ono the oorder lib.

Reply:
I will revert back all changes in `organization`